openedx_events/bridge/tests/test_utilities.py

```diff
@@ -0,0 +1,117 @@
+"""
+Utility methods and classes for testing AvroAttrsBridge
+"""
+import io
+import re
+
+import attr
+import fastavro
+
+from openedx_events.bridge.avro_attrs_bridge_extensions import AvroAttrsBridgeExtention
+from openedx_events.tooling import OpenEdxPublicSignal
+
+
+def create_simple_signal(data_dict):
+    """
+    Create a basic OpenEdxPublicSignal with init_data = data_dict
+    :param data_dict: Description of attributes passed to the signal
```

Comment:
I think we use Google Style docstrings: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings. Looks like you changed some methods above as well?